src/init.ts

```diff
@@ -343,24 +344,24 @@ export class NightwatchInit {
 
     const testsJsSrc: string = path.join(this.otherInfo.tsOutDir || '', answers.testsLocation || '');
     if (testsJsSrc !== '.') {
-      if (answers.testsLocation === answers.examplesLocation && answers.language === 'js' && answers.runner !== 'cucumber') {
+      if (answers.testsLocation === answers.examplesLocation && answers.language === 'js' && answers.runner !== Runner.Cucumber) {
         // examples are being put as a boilerplate in testsLocation with main tests in
-        // 'specs' sub-directory (only done for JS-Nightwatch and JS-Mocha).
-        src_folders.push(path.join(testsJsSrc, 'specs'));
+        // EXAMPLE_TEST_FOLDER sub-directory (only done for JS-Nightwatch and JS-Mocha).
+        src_folders.push(path.join(testsJsSrc, EXAMPLE_TEST_FOLDER));
       } else {
         src_folders.push(testsJsSrc);
       }
```

Comment:
Based on the answer to the previous comment, changes would be required here as well. The above changes are correct for the older approach of putting examples in tests directory (if empty). But if we are to go forward with the new approach (always putting examples in `nightwatch` directory), the above will change to just:
```js
if (testsJsSrc !== '.') {
  src_folders.push(testsJsSrc);
}
```